### DISCUSSION TITLE
RNode for Canary One by canaryradio.io

### DISCUSSION THREAD

**Discussion by SignalMedic** [markqvist/Reticulum#574] [Help & Questions]
I am starting to investigate what it will take to build Rnode firmware for the Canary One board. If anyone has advice I would appreciate it.

**Comment by faragher** [comment]
The nRF52840 and SX1262 are supported, so I'd start with the RAK4631 board from https://github.com/liberatedsystems/RNode_Firmware_CE

You'll likely need to crack it open and trace pins to properly define the board, but that's your best bet.
